src/flb_record_accessor.c

```diff
@@ -453,10 +453,20 @@ static flb_sds_t ra_translate_keymap(struct flb_ra_parser *rp, flb_sds_t buf,
     }
     else if (v->type == FLB_RA_INT) {
         len = snprintf(str, sizeof(str) - 1, "%" PRId64, v->val.i64);
+        /* We need to check size is not above str length */
+        if (len >= 32) {
```

Comment:
32 = sizeof(str).
It is better to use `sizeof(str)`.

Reply:
What an input does match this case?
int64_t is `-9223372036854775808` ~ `9223372036854775807`.

The length of `-9223372036854775808`  is smaller than 32.

Reply:
You're right. The length check only needs to be therefor floats. Thanks!